examples/games/game_menu.rs

```diff
@@ -530,12 +529,12 @@ mod menu {
         commands
             .spawn((
                 NodeBundle {
-                    style: Style {
-                        margin: UiRect::all(Val::Auto),
-                        flex_direction: FlexDirection::Column,
+                    spacing: Spacing::AUTO_MARGIN,
+                    layout: Layout::Flex(FlexLayout {
+                        direction: Direction::ColumnReverse,
```

Comment:
```suggestion
                        direction: Direction::Column,
```
This was swapped in the y-axis flip PR.

Reply:
Fixed